src/query/functions/temporal/aggregation.go

```diff
@@ -192,3 +220,62 @@ func sumAndCount(values []float64) (float64, float64) {
 
 	return sum, count
 }
+
+func makeQuantileOverTimeFn(scalar float64) aggFunc {
+	return func(values []float64) float64 {
+		valuesSlice := make(valsSlice, 0, len(values))
+		for _, v := range values {
+			valuesSlice = append(values, v)
```

Comment:
What is this loop trying to achieve? This seems to be overwriting valuesSlice on each iteration with the whole of values + the current value from values. You end up with just the values slice + the very last value in values in valuesSlice. Is that what you want?

Reply:
+1--looks like it should be `valuesSlice`? You could probably also use `copy` for this (assuming that's the goal)